src/euphorie/client/browser/risk.py

```diff
@@ -261,18 +261,23 @@ def __call__(self):
         else:
             self._prepare_risk()
             if self.is_custom_risk:
-                template = ViewPageTemplateFile(
-                    "templates/risk_identification_custom.pt"
-                ).__get__(
-                    self, ""
-                )  # noqa
                 next = FindNextQuestion(
                     self.context, self.context.session, filter=self.question_filter
                 )
                 self.has_next_risk = next or False
-            else:
-                template = self.template
-            return template()
+            return self.template()
```

Comment:
If my previous suggestions are accepted this can be:
```
return self.custom_risk_template.__get__(self, "")() 
```
Actually I think this might also work:
```
return self.custom_risk_template() 
```
Not sure what `.__get__(self, "")` does...